documentation/modules/appendix_crds.adoc

```diff
@@ -1285,6 +1288,52 @@ Used in: xref:type-ContainerTemplate-{context}[`ContainerTemplate`]
 |The environment variable value.
 |====
 
+[id='type-TieredStorageCustom-{context}']
+= `TieredStorageCustom` schema reference
+
+Used in: xref:type-KafkaClusterSpec-{context}[`KafkaClusterSpec`]
+
+xref:type-TieredStorageCustom-schema-{context}[Full list of `TieredStorageCustom` schema properties]
+
+include::../api/io.strimzi.api.kafka.model.kafka.tieredstorage.TieredStorageCustom.adoc[leveloffset=+1]
+
+[id='type-TieredStorageCustom-schema-{context}']
+== `TieredStorageCustom` schema properties
+
+
```

Comment:
too many blank lines

Reply:
This is generated file. So I suspect the problems is not in this PR.

Reply:
you are right :-/